### PR TITLE
Create assignment2

### PR DESCRIPTION
**Pull request by alexandracalota**

### CODE REVIEW

File: assignment2 [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	private Node leftSon;
+	private Node rightSon;
+
+	public Node(int key) {
+		this.key = key;
+		leftSon = null;
+		rightSon = null;
+	}
+
+	public void insertNode(int key, int parentKey, String position) {
```
Comment (kkasiacodeu) [inline assignment2]: if you only allow "L" and "R" values here you can use char instead of String.

File: assignment2 [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+			return root;
+		else if (left != null)
+			return left;
+		else
+			return right;
+	}
+	
+	
+	/******************** Question 1 ********************/
+	public static Boolean auxQ1(Stack<Node> parents, int key, Node root) {
```
Comment (kkasiacodeu) [inline assignment2]: what does the name if this method stand for? Could you rename it so that it says what it does?

Comment (kkasiacodeu) [inline assignment2]: You don't have to wrap the primitive boolean into a Boolean object in the return type.

**Review by kkasiacodeu — CHANGES_REQUESTED** [review]
Nice work! :) Good algorithms, only a few minor comments to address.

File: assignment2 (hunk 2 above)
Comment (alexandracalota) [reply assignment2]: I changed the name so it is suggestive.

Comment (alexandracalota) [reply assignment2]: I changed the return type to boolean.

File: assignment2 (hunk 1 above)
Comment (alexandracalota) [reply assignment2]: Yes, that would be simpler. I change the type of the argument.

**Pull request merged**